README.md

```diff
@@ -1,2 +0,0 @@
-# EPAM_GO
```

Comment:
Anyway here is no need to remove README.md, I guess=)

Reply:
Yeah, that was some silly mistake with Git...